### PR TITLE
bios lock cherry-pick

### PR DESCRIPTION
**Pull request by Stojek139808**

### CODE REVIEW

File: src/vendorcode/intel/Makefile.inc [hunk 1]
```diff
@@ -14,6 +14,7 @@ CPPFLAGS_x86_32 += -I$(src)/vendorcode/intel/edk2/UDK2017/MdePkg/Include/Ia32
 CPPFLAGS_x86_64 += -I$(src)/vendorcode/intel/edk2/UDK2017/MdePkg/Include/X64
 CPPFLAGS_common += -I$(src)/vendorcode/intel/edk2/UDK2017/MdePkg/Include
 CPPFLAGS_common += -I$(src)/vendorcode/intel/edk2/UDK2017/IntelFsp2Pkg/Include
+CPPFLAGS_common += -I$(src)/vendorcode/intel/edk2/UDK2017/MdeModulePkg/Include
```
Comment (mkopec) [inline src/vendorcode/intel/Makefile.inc:17]: On Alder Lake I also had to add the VariableFormat.h file to `edk2-stable202005` bindings. See commit https://github.com/Dasharo/coreboot/commit/1f08d9bc633fc862c61dbc76859d879fe136b90c, should be able to cherry-pick it here.

Comment (mkopec) [reply src/vendorcode/intel/Makefile.inc:17]: Looks like that will also be needed on Elkhart Lake.

Comment (Stojek139808) [reply src/vendorcode/intel/Makefile.inc:17]: Cherry-picked [this commit](https://github.com/Dasharo/coreboot/commit/e70230817c08738a8ee0ca3d5d5af62986cf9933) instead, it's the same change, just had to find it in the tree.

**Review by mkopec — APPROVED** [review]

**Comment by mkopec** [comment]
@Stojek139808 needs a rebase

**Comment by Stojek139808** [comment]
> @Stojek139808 needs a rebase

Done.

File: payloads/external/edk2/Makefile [hunk 2]
```diff
@@ -253,6 +253,9 @@ prep: $(EDK2_PATH) clean checktools logo lan_rom ipxe_rom
 $(WORKSPACE)/Build/UefiPayloadPkgX64/$(RELEASE_STR)_COREBOOT/FV/UEFIPAYLOAD.fd: \
 	prep print
 	cd $(WORKSPACE); \
+		echo -n "Building iPXE... "; \
+		cd $(PACKAGES_PATH)/UefiPayloadPkg/NetworkDrivers; \
+		./build-ipxe.sh; \
```
Comment (miczyg1) [inline payloads/external/edk2/Makefile:258]: I just ditched this and enabled building iPXE through coreboot build system. Please do not include this commit.

**Review by miczyg1 — CHANGES_REQUESTED** [review]

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**